### PR TITLE
Update all-in-one deployment docs

### PR DESCRIPTION
**Pull request by dushaniw**
## Purpose
Fix https://github.com/wso2/docs-apim/issues/925

### CODE REVIEW

File: en/docs/install-and-setup/deploying-wso2-api-manager/production-deployment-guidelines.md [hunk 1]
```diff
@@ -54,13 +60,12 @@ Prior to installing any WSO2 Carbon-based product, it is necessary to have the a
...
-#### 
-Environment compatibility
+### Environment compatibility
 
 -   By default, WSO2 products are installed with **OpenJDK** , which allows you to run the product as soon as it is installed.
 
-        !!! tip
-    To use a different JDK, point the **`JAVA_HOME           `** environment variable to the new JDK. Make sure your JDK version is [compatible with the WSO2 product](https://docs.wso2.com/display/compatibility/Tested+Operating+Systems+and+JDKs) .
+    !!! tip
+        To use a different JDK, point the **`JAVA_HOME           `** environment variable to the new JDK. Make sure your JDK version is [compatible with the WSO2 product](https://docs.wso2.com/display/compatibility/Tested+Operating+Systems+and+JDKs) .
```
Comment (tgtshanika) [inline en/docs/install-and-setup/deploying-wso2-api-manager/production-deployment-guidelines.md]: Shall we point this to {base_path}/install-and-setup/product-compatibility/#product-compatibility

File: en/docs/install-and-setup/deploying-wso2-api-manager/single-node/configuring-a-single-node.md [hunk 2]
```diff
@@ -6,20 +6,79 @@ This page walks you through how to manually configure and deploy WSO2 API Manage
...
 ----------------------------
 
-## Configure URLs that are used to expose APIs
+## Step 1 - Create a SSL Certificate
 
-Add this hostname when you configure environments in the `<API-M_HOME>/repository/conf/deployment.toml` file. Update the endpoints with your chosen hostname as shown below. 
-In this case, let's use `gw.wso2.com` as the hostname:
+!!! info   
+    All WSO2 products are by default shipped with a keystore file and truststore file stored in the 
+    `<PRODUCT_HOME>/repository/resources/security/ directory`. The default keystore that is shipped with a WSO2 product,
```
Comment (tgtshanika) [inline en/docs/install-and-setup/deploying-wso2-api-manager/single-node/configuring-a-single-node.md]: `<PRODUCT_HOME>/repository/resources/security/` directory.

File: en/mkdocs.yml [hunk 3]
```diff
@@ -534,6 +534,6 @@ extra:
     - type: linkedin
       link: https://www.linkedin.com/company/wso2
   site_version: 3.1.0
-  base_path: https://apim.docs.wso2.com/en/next
+  base_path: http://localhost:8000
```
Comment (tharindu1st) [inline en/mkdocs.yml]: revert the change

Comment (dushaniw) [reply en/mkdocs.yml]: fixed

File: en/docs/install-and-setup/deploying-wso2-api-manager/single-node/configuring-a-single-node.md (hunk 2 above)
Comment (dushaniw) [reply en/docs/install-and-setup/deploying-wso2-api-manager/single-node/configuring-a-single-node.md]: Fixed
